### DISCUSSION TITLE
Comments sidebar doesn't work in edit mode

### DISCUSSION THREAD

**Discussion by deighvebee** [GitbookIO/community#200] [0. General bugs and questions] [answered]
When I'm in edit mode, I cannot see the list of comments in the sidebar. That's unfortunate, because when I'm trying to resolve them that means I need to have a read-only version of the content open in another window that's separate from my active editor. I can't imagine that's working by design.

**Comment by aikoii** [comment, marked as answer]
Hi @deighvebee,

Thank you for reaching out about this and sharing your feedback!

This is how comments work now, i.e. they are not carried over from the main content to an edit mode/change request. That said, I'm happy to let you know we have a dedicated team currently working on this flow, and what you described here is exactly the kind of thing we're building 🙂.

We will keep you informed when this will be released!

**Reply by deighvebee** [reply]
That's great to hear, thank you!

**Comment by clairegoldschmidt** [comment]
hey just to add to this I've found adding comments in edit mode are wiped after a merge has been completed.   No history of the comments either.
